Makefile

```diff
@@ -225,6 +233,14 @@ ifdef ANALYTICS_MIXPANEL_TOKEN
 		-t $(BUILD_TEMPORARY_DIRECTORY)
 endif
 
+ifdef DISABLE_UPDATES
+	./scripts/build/jq-insert.sh \
+		-p "updates.enabled" \
+		-v "false" \
+		-f $@/package.json \
+		-t $(BUILD_TEMPORARY_DIRECTORY)
+endif
```

Comment:
Very nice :+1: 
Should this `DISABLE_UPDATES` variable be documented somewhere?